src/Platforms/OraclePlatform.php

```diff
@@ -478,6 +478,12 @@ public function getCreateViewSQL($name, $sql)
      */
     public function getDropViewSQL($name)
     {
+        $id = new Identifier($name);
+
+        if(!$id->isQuoted() && strtoupper($name) !== $name) {
+            $name = $this->quoteIdentifier($name);
+        }
```

Comment:
This block is repeated many times… wouldn't it be worth extracting it in a private method with a meaningful name that hints at what it does?

Reply:
Done